cargo-{help,version}: add page (#10712)
tldr-pages/tldr: pages/common/cargo-help.md

# cargo help

> > Display help information about the Rust project manager.
> > More information: <https://doc.rust-lang.org/cargo/commands/cargo-help.html>.

Reviewer: ```suggestion
> Display help on `cargo` and its subcommands.
> More information: <https://doc.rust-lang.org/cargo/commands/cargo-help.html>.
```

- Display general help:

`cargo help`

> 
> - Display help for the `cargo help` subcommand:
> 
> `cargo help --help`

Reviewer: ```suggestion
```
